azure-pipelines.yml

```diff
@@ -874,4 +906,19 @@ jobs:
     inputs:
       PathtoPublish: $(Build.ArtifactStagingDirectory)/zip
       ArtifactName: zipv2
+
+  - task: SdtReport@2
+    inputs:
+      GdnExportAllTools: true
+      GdnExportHtmlFile: true
+  - task: PublishSecurityAnalysisLogs@3
+    inputs:
+      ArtifactName: 'CodeAnalysisLogs'
+      ArtifactType: 'Container'
+      AllTools: true
+      ToolLogsNotFoundAction: 'Standard'
+  - task: PostAnalysis@2
```

Comment:
Only suggestion I would have is, I see the build fails but doesn't show me reason of failure. someone new would struggle to find why build fails.  can we check if there is way for the post analysis to report that code sign validation failed or since we use this only for code sign , we can add only code sign validation to SdtReport@2 instead of all tools. Just an idea/suggestion. Thank you. 

![image](https://user-images.githubusercontent.com/105889062/223774998-416526ac-4cfd-40e7-bc6f-d9fad2f2b33b.png)

Reply:
Needs more reading about guardian break, currently we are trying to keep this simple.